presto-client/src/main/java/com/facebook/presto/client/StatementStats.java

```diff
@@ -97,6 +103,12 @@ public String getState()
         return state;
     }
 
+    @JsonProperty
+    public boolean isWaitingForPrerequisites()
```

Comment:
I am maintaining symmetry in this class. We already have `isQueued` despite the state being there and hence added this. One of the reasons why I feel this might have been done is that state is a String and not an enum - and hence this could provide people an easy way without figuring out the strings if the query has started or not.